src/status_im2/contexts/quo_preview/list_items/dapp.cljs

```diff
@@ -0,0 +1,61 @@
+(ns status-im2.contexts.quo-preview.list-items.dapp
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.common.resources :as resources]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "State"
```

Comment:
You can omit `:label`, it will be autogenerated from `:key`

Reply:
nice! done.